### PR TITLE
Added missing colon in for loop.

### PR DESCRIPTION
**Pull request by andre-c-andersen**

### CODE REVIEW

File: softmax.py [hunk 1]
```diff
@@ -101,7 +101,7 @@ def main(argv=None):
         tf.initialize_all_variables().run()
         
         # Iterate and train.
-        for step in xrange(num_epochs * train_size // BATCH_SIZE)
+        for step in xrange(num_epochs * train_size // BATCH_SIZE):
```
Comment (SanjithKanagavel) [inline softmax.py:104]: Please change these ones too for Python 3.6:
In **softmax.py** :
1. `xrange` has been changed to `range`. 
2. `file(filename):` to `open(filename):`

In **plot_boundary_on_data.py** :
1.   `plt.scatter(X[:,0], X[:,1], c=Y[:,1], s=50,cmap=colors.ListedColormap(['orange', 'blue']))` to
    `plt.scatter(np.squeeze(np.asarray(X[:,0])),  np.squeeze(np.asarray(X[:,1])), c=Y[:,1], s=50,cmap=colors.ListedColormap(['orange', 'blue']))`
